netty/src/main/java/io/grpc/netty/Utils.java

```diff
@@ -84,6 +88,9 @@
   public static final AttributeKey<SSLSession> SSL_SESSION_ATTR_KEY =
       AttributeKey.valueOf(SSLSession.class, "ssl-session");
 
+  @VisibleForTesting
+  static boolean validateHeaders = false;
```

Comment:
Realize that the very presence of this field (that it isn't hard-coded) may impact the benchmark.